src/components/Dropdown/index.js

```diff
@@ -66,9 +95,15 @@ class Dropdown extends Component {
 Dropdown.propTypes = {
   children: propTypes.element,
   options: propTypes.arrayOf(propTypes.object).isRequired,
+  // eslint-disable-next-line react/forbid-prop-types
+  subOptions: propTypes.object,
```

Comment:
shape({})

Reply:
@masteralx we cannot use shape this, because its object, contains balances, which keys is wallets addresses